tests/sles4sap/netweaver_ascs.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's SLES4SAP openQA tests
+
+# Summary: Checks NetWeaver's ASCS installation as performed by sles4sap/nw_ascs_install
+# Requires: sles4sap/nw_ascs_install
+# Maintainer: Alvaro Carvajal <acarvajal@suse.de>
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    my $pscmd = "ps auxw | grep ASCS | grep -vw grep";
+    $pscmd = "$pscmd | wc -l ; $pscmd";
+
+    x11_start_program('xterm');
+    assert_screen('xterm');
+
+    # The SAP Admin user 'qadadm' was set in sles4sap/nw_ascs_install
+    my $sapadmin = 'qadadm';
```

Comment:
if you share some data between modules , probably it worth to do it via variable.
Use set_var function to init sapadmin in nw_ascs_install and use get_var here this will give you more safety and flexibility

Reply:
Will apply that change. Thanks

Reply:
Done.